### PR TITLE
#135 add a reference to matrix.xsd from the generated matrix.xml

### PR DESCRIPTION
**Pull request by ilyakharlamov**

### CODE REVIEW

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#288](https://github.com/yegor256/jpeek/issues/288) is now in scope, role is `REV`

**Comment by paulodamaso** [comment]
@ilyakharlamov Please add a description to the pull request telling what you've done and referencing to the issue

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) everybody who has role `REV` is banned at [#288](https://github.com/yegor256/jpeek/issues/288); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/test/java/org/jpeek/MatrixTest.java [hunk 1]
```diff
@@ -40,18 +41,39 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  */
 public final class MatrixTest {
+    /**
+     * Xml as a string.
+     */
+    private String xmltext;
```
[the remark this answers is not in the document]
Comment (ilyakharlamov) [reply src/test/java/org/jpeek/MatrixTest.java]: @paulodamaso renamed

**Comment by paulodamaso** [comment]
@ilyakharlamov Why have you added more things to this pull request?

**Comment by ilyakharlamov** [comment]
@paulodamaso Not sure why it keeps pulling the unrelated stuff from my master. Do I need to create a separate branch for every pull request?

**Comment by 0crat** [comment]
Job `gh:yegor256/jpeek#288` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#288](https://github.com/yegor256/jpeek/issues/288) is now out of scope

**Pull request closed without merging**